app/models/create_comment.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComment < ApplicationRecord
+  belongs_to :author, class_name: 'CreateUser', foreign_key: :author_id
+  has_many :create_comments, foreign_key: :create_posts_id
+
+  def update_comments_counter
+    post.increment(:comments_counter)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Just like I have recommended in the Post counter. We can have the same for the Comment counter as well:
`after_save :update_comments_counter`